### ISSUE TITLE
Issuing a GET request with NTLM authentication [beta] and HTTPS endpoint closes the application

### ISSUE THREAD

**Issue by Alexei000** [a85/POSTMan-Chrome-Extension#5768]
Please read through the [guidelines](https://github.com/postmanlabs/postman-app-support#guidelines-for-reporting-issues) before creating a new issue.

**Describe the bug**
I am trying to perform a GET request against a localhost https url that uses NTLM authentication. I am using the default settings except for SSL certificate verification (turned off). Doing this closes the application after about 1s of doing nothing noticeable. I cannot see any event within Event Viewer (any other place to look for useful logging info?).

**To Reproduce**
1. Make sure that SSL certificate verification is turned off. Also Yes, disable retrying the request option must remain unchecked (otherwise a 401 is received)
2. Create a request to localhost (e.g. https://localhost/....)
3. Execute the request
4. Postman closes without any error

**Expected behavior**
Postman executes the request and returns the data

**Screenshots**

**App information (please complete the following information):**
 - Native App
 - Postman Version [e.g. 6.0.1]
 - OS: Windows 10 Professional x64 (10.0.17763)

**Additional context**
The same steps work both in Chrome browser and Postman Chrome extension. The headers as received from Postman Chrome extension which handle this request correctly:

```
content-length →112
content-type →application/json; charset=utf-8
date →Sat, 12 Jan 2019 19:51:14 GMT
persistent-auth →true
server →Microsoft-IIS/10.0
x-powered-by →ASP.NET
```

**Comment by numaanashraf** [comment]
@Alexei000 Thanks for reporting this. Could you check for any errors in the app logs? (View > Developer > Show Logs)

**Comment by Alexei000** [comment]
Yes, I have managed to isolate the error in the logs. Please, check it below:

```
[19764][1547834883120][main][info]["Main\\x7eAppEvents - Received booted event for process requester"]
[19764][1547834888758][main][error]["Main\\x7ehandleUncaughtError - Uncaught errors",{"name":"TypeError","message":"First argument must be a string, Buffer, ArrayBuffer, Array, or array-like object.","stack":"TypeError: First argument must be a string, Buffer, ArrayBuffer, Array, or array-like object.\n    at Function.Buffer.from (buffer.js:179:11)\n    at new Buffer (buffer.js:154:17)\n    at Object.createType3Message (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\httpntlm\\ntlm.js:172:19)\n    at Object.post (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\lib\\authorizer\\ntlm.js:140:44)\n    at Object.module.exports (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\lib\\runner\\request-helpers-postsend.js:31:21)\n    at C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\node_modules\\async\\dist\\async.js:241:31\n    at C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\node_modules\\async\\dist\\async.js:1135:9\n    at replenish (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\node_modules\\async\\dist\\async.js:1011:17)\n    at C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\node_modules\\async\\dist\\async.js:1016:9\n    at _asyncMap (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\node_modules\\async\\dist\\async.js:1133:5)\n    at C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\node_modules\\async\\dist\\async.js:1219:16\n    at C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\node_modules\\async\\dist\\async.js:1046:16\n    at Object.<anonymous> (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\node_modules\\async\\dist\\async.js:240:20)\n    at Object.<anonymous> (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\node_modules\\async\\dist\\async.js:74:12)\n    at Object.applyEachSeries (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\node_modules\\async\\dist\\async.js:245:23)\n    at C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\lib\\runner\\extensions\\http-request.command.js:182:31\n    at complete (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\lib\\requester\\requester.js:109:24)\n    at Request._callback (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-runtime\\lib\\requester\\requester.js:178:13)\n    at Request.self.callback (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-request\\request.js:247:22)\n    at emitTwo (events.js:125:13)\n    at Request.emit (events.js:213:7)\n    at Request.<anonymous> (C:\\Users\\Alex\\AppData\\Local\\Postman\\app-6.7.1\\resources\\app.asar\\node_modules\\postman-request\\request.js:1230:10)"}]
```

**Comment by steigerm** [comment]
Seems that I have a similar situation (#6231).
Difference is: NO SSL just http://localhost

**Comment by vikiCoder** [comment]
Hi everyone,

This issue is fixed in the latest release of Postman App (v7.14.0). Please update your Postman app to the latest version and let us know if you are still facing any issues regarding this.
